addons/payment_stripe_sca/__manifest__.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+#  ____________________
+# /                    \
+# |   DO NOT FORWARD   |
+# |    PORT FURTHER    |
+# |     THAN SAAS-13   |
```

Comment:
SAAS-13? 🤨

Reply:
@mreficent 
 yes, I will create different PR for Saas-14 and later version :)